projects/canopy-test-app/src/app/app.component.html

```diff
@@ -306,6 +306,14 @@
               <lg-radio-button value="blue">Blue</lg-radio-button>
             </lg-radio-group>
 
+            <lg-filter-group formControlName="filter">
+              Test filter group
+              <lg-radio-button value="red">Red</lg-radio-button>
```

Comment:
we can add the multiple selectors here too but it will not add any functionality that is why I left it out but I'm ok with adding them if it adds a visual appeal.

Reply:
it is purely visual appeal, and I guess maybe readability, shall we await a second opinion on the next reviewer?

Reply:
I will add it as it makes more sense having `lg-filter-button` in `lg-filter-group` 👍

Reply:
Done.

Reply:
I agree with the readability side of things 👍